crates/chia-wallet/src/proof.rs

```diff
@@ -4,7 +4,7 @@ use clvm_traits::{FromClvm, ToClvm};
 #[derive(Debug, Clone, PartialEq, Eq, ToClvm, FromClvm)]
 #[cfg_attr(fuzzing, derive(arbitrary::Arbitrary))]
 #[clvm(untagged, tuple)]
-pub enum Proof {
+pub enum MaybeEveProof {
```

Comment:
this change seems unrelated. what prompted it?

Reply:
Renamed for clarity, since most of the time it's going to be a LineageProof not an EveProof (only during launching is it Eve)

Reply:
I reverted the naming and use of generics, in favor of using match instead